airbyte-webapp/src/components/JobItem/JobItem.tsx

```diff
@@ -76,16 +64,8 @@ const JobItem: React.FC<IProps> = ({ shortInfo, ...props }) => {
   });
 
   return (
-    <Item isFailed={isFailed} ref={scrollAnchor}>
-      <MainInfo
-        shortInfo={shortInfo}
-        isOpen={isOpen}
-        isFailed={isFailed}
-        isPartialSuccess={isPartialSuccess}
-        onExpand={onExpand}
-        job={jobMeta}
-        attempts={isJobEntity(props) ? props.job.attempts : undefined}
-      />
+    <Item isFailed={didSucceed} ref={scrollAnchor}>
```

Comment:
I'm pretty sure this one is causing the light red color on the hover
```suggestion
    <Item isFailed={!didSucceed} ref={scrollAnchor}>
```